bio/minimap2/aligner/wrapper.py

```diff
@@ -45,6 +46,7 @@
     "(minimap2"
     " -t {snakemake.threads}"
     " {extra} "
+    " -O {gap_opening} "
```

Comment:
Currently, the `-O ` always gets added to the command, even if no `gap_opening` is specified. So this rather needs to become:
```suggestion
    "{gap_opening} "
```

And then the `-O ` needs to be prepended to gap_opening above, if it is none-empty. This should be possible with something like:
```
if gap_opening:
    gap_opening = f"-O {gap_opening}"
```
```